grouper/graph.py

```diff
@@ -179,9 +179,10 @@ def user_indexify(data):
                         "created_on": str(key.created_on),
                         "tags": [tag.name for tag in public_key_tags.get(key.id, [])],
                         "permissions": lambda: list(set([(perm.name, perm.argument) for perm in
-                                                   get_public_key_permissions(session,
-                                                        PublicKey.get(session, id=key.id)
-                                                    )
+                                                   get_public_key_permissions(
+                                                       session,
+                                                       PublicKey.get(session, id=key.id),
+                                                   )
```

Comment:
changes to unrelated files for syntax purity are best separated out. 

`git add -p` ftw